spec/models/reports/prescription_spec.rb

```diff
@@ -0,0 +1,157 @@
+require "rails_helper"
+
+# Shorthand for adding prescription drugs, since we do it a lot in this spec
+def add_drug(patient, name, dosage, time, do_not_delete: false)
+  # emulate deletion of existing drug like on mobile app
+  unless do_not_delete
+    patient.prescription_drugs
+      .where(name: name, is_deleted: false)
+      .update(is_deleted: true, device_updated_at: time)
+  end
+
+  # create new drug
+  FactoryBot.create(:prescription_drug,
+    patient: patient,
+    facility: patient.registration_facility,
+    device_created_at: time,
+    device_updated_at: time,
+    name: name,
+    dosage: dosage)
+end
+
+RSpec.describe Reports::Prescription, {type: :model, reporting_spec: true} do
+  describe "Associations" do
+    it { should belong_to(:patient) }
+  end
+
+  around do |example|
+    Timecop.freeze("June 30 2021 5:30 UTC") do
```

Comment:
Use the same date as in the `reporting_helper`. Comment is incorrect.

Reply:
Fixed in 119553be5.